### DISCUSSION TITLE
[Algolia] next主题使用Algolia搜索引擎

### DISCUSSION THREAD

**Discussion by Angel0726** [tangly1024/NotionNext#2089] [Ideas]
**描述bug**
Algolia搜索引擎部署成功。heo主题可以使用，但是next主题不能使用

博客地址：https://www.vabc.eu.org/
**复现步骤**
![image](https://github.com/tangly1024/NotionNext/assets/32816780/00f6a9a1-18a5-42b4-a43c-7e8c752cd98c)

![image](https://github.com/tangly1024/NotionNext/assets/32816780/6fdc1563-5bbd-42fc-8fb6-ff55edc1c877)

**期望的正常结果**
希望按这个步骤，正常操作结果是什么

**截图**
相关的页面，应该的结果

**环境**

- 操作系统: [例如. iOS, Android, macOS, windows]
- 浏览器 [例如. chrome, safari, firefox]
- NotionNext版本 [e.g. 3.13.6]
- 主题 [例如. hexo]

**补充说明**
与问题相关的其它说明

**Comment by tangly1024** [comment]
暂时只支持HEO主题，后面会陆续加入到其它主题

**Comment by Angel0726** [comment]
这个功能，可以优先适配一下其他主题吗？
